### PR TITLE
HybridConnectivity SDK Update

### PR DESCRIPTION
**Pull request by alexmahonic**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Review by jsquire — DISMISSED** [review]
@alexmahonic :

Your account lacks the required public GitHub organizations and permissions required of an internal contributor.  Please review the Azure SDK onboarding documentation and use the associated Teams channel for support.
- [Azure SDK onboarding](https://eng.ms/docs/products/azure-developer-experience/onboard/requirements) _(Microsoft internal)_
- [Azure SDK onboarding assistance](https://teams.microsoft.com/l/channel/19%3ade3fce22c2994be18cac50502c55f717%40thread.skype/Azure%2520SDK%2520Onboarding?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) _(Microsoft internal)_

You can verify the state of your account by running the [Validate-AzsdkCodeOwner](https://github.com/Azure/azure-sdk-tools/blob/main/tools/github/scripts/Validate-AzsdkCodeOwner.ps1) script from the Azure SDK tools repository.

Please also be sure to add yourself to [CODEOWNERS](https://github.com/Azure/azure-sdk-for-net/blob/main/.github/CODEOWNERS) for this library, if you will be maintaining it going forward.

**Comment by alexmahonic** [comment]
> @alexmahonic :
> 
> Your account lacks the required public GitHub organizations and permissions required of an internal contributor. Please review the Azure SDK onboarding documentation and use the associated Teams channel for support.
> 
> * [Azure SDK onboarding](https://eng.ms/docs/products/azure-developer-experience/onboard/requirements) _(Microsoft internal)_
> * [Azure SDK onboarding assistance](https://teams.microsoft.com/l/channel/19%3ade3fce22c2994be18cac50502c55f717%40thread.skype/Azure%2520SDK%2520Onboarding?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) _(Microsoft internal)_
> 
> You can verify the state of your account by running the [Validate-AzsdkCodeOwner](https://github.com/Azure/azure-sdk-tools/blob/main/tools/github/scripts/Validate-AzsdkCodeOwner.ps1) script from the Azure SDK tools repository.
> 
> Please also be sure to add yourself to [CODEOWNERS](https://github.com/Azure/azure-sdk-for-net/blob/main/.github/CODEOWNERS) for this library, if you will be maintaining it going forward.

Hi @jsquire, thanks for you comments.

- I've raise another PR https://github.com/Azure/azure-sdk-for-net/pull/46977 to add myself as the codeowner of it. Please review.
- The  [Validate-AzsdkCodeOwner](https://github.com/Azure/azure-sdk-tools/blob/main/tools/github/scripts/Validate-AzsdkCodeOwner.ps1) script indicates that I'm not of the required organizations Microsoft and Azure while I'm already in those 2 orgs and the membership visibilities are set to Public. I'm not sure if I need to honor the result of the script. Please advise. 
image
image
- I didn't request the write access permission because I thought it was not necessary for me according to this [guide](https://eng.ms/docs/products/azure-developer-experience/onboard/access#write-access). Please confirm if I need to.

**Comment by jsquire** [comment]
@alexmahonic :  You do need write permissions to contribute to the repository and to register as a code owner.  I see the text that is confusing in the onboarding guide.   I'll work on getting that updated to be more clear.

**Comment by alexmahonic** [comment]
> @alexmahonic : You do need write permissions to contribute to the repository and to register as a code owner. I see the text that is confusing in the onboarding guide. I'll work on getting that updated to be more clear.

Thanks, I followed the guide and requested the write permission.

File: sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/ServiceConfigurationResourceData.cs [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+            ResourceId = resourceId;
+            Port = port;
+            ProvisioningState = provisioningState;
+            _serializedAdditionalRawData = serializedAdditionalRawData;
+        }
+
+        /// <summary> Name of the service. </summary>
+        public ServiceName? ServiceName { get; set; }
+        /// <summary> The resource Id of the connectivity endpoint (optional). </summary>
+        public string ResourceId { get; set; }
```
Comment (archerzz) [inline sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/ServiceConfigurationResourceData.cs:81]: What's the format of this property value?
- If it's a GUID, use type [Guid](https://learn.microsoft.com/en-us/dotnet/api/system.guid?view=net-6.0). See example [here](https://github.com/Azure/azure-sdk-for-net/blob/bd953fc7a97bf617c892ce6314b50126b4021a4f/sdk/maps/Azure.ResourceManager.Maps/src/autorest.md?plain=1#L25).
- If it's a resource identifier, use type [ResourceIdentifier](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/ResourceIdentifier.cs). See example [here](https://github.com/Azure/azure-sdk-for-net/blob/bd953fc7a97bf617c892ce6314b50126b4021a4f/sdk/search/Azure.Search.Documents/src/autorest.md?plain=1#L106-L115).

File: sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/AwsCloudProfile.cs [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+            _serializedAdditionalRawData = serializedAdditionalRawData;
+        }
+
+        /// <summary> Initializes a new instance of <see cref="AwsCloudProfile"/> for deserialization. </summary>
+        internal AwsCloudProfile()
+        {
+        }
+
+        /// <summary> Account id for the AWS account. </summary>
+        public string AccountId { get; set; }
```
Comment (archerzz) [inline sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/AwsCloudProfile.cs:78]: Is it a GUID?

File: sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/InventoryResource.cs [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+
+namespace Azure.ResourceManager.HybridConnectivity
+{
+    /// <summary>
+    /// A Class representing an InventoryResource along with the instance operations that can be performed on it.
+    /// If you have a <see cref="ResourceIdentifier"/> you can construct an <see cref="InventoryResource"/>
+    /// from an instance of <see cref="ArmClient"/> using the GetInventoryResource method.
+    /// Otherwise you can get one from its parent resource <see cref="SolutionConfigurationResource"/> using the GetInventoryResource method.
+    /// </summary>
+    public partial class InventoryResource : ArmResource
```
Comment (archerzz) [inline sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/InventoryResource.cs:23]: The name is too generic. Consider appending RP prefix or renaming it.
- use `prepend-rp-prefix` to automatically append PR name. See example [here](https://github.com/Azure/azure-sdk-for-net/blob/176fd476588e8e61ebce747d66714cb39947cae5/sdk/azurestackhci/Azure.ResourceManager.Hci/src/autorest.md?plain=1#L52-L57).
- use `rename-mapping` to rename a model. See example [here](https://github.com/Azure/azure-sdk-for-net/blob/176fd476588e8e61ebce747d66714cb39947cae5/sdk/azurestackhci/Azure.ResourceManager.Hci/src/autorest.md?plain=1#L58-L76)

File: sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/CloudNativeType.cs [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.ResourceManager.HybridConnectivity.Models
+{
+    /// <summary> Cloud Native Type enum. </summary>
+    public readonly partial struct CloudNativeType : IEquatable<CloudNativeType>
```
Comment (archerzz) [inline sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/CloudNativeType.cs:14]: The name is too generic. Consider appending RP prefix or renaming it.
- use `prepend-rp-prefix` to automatically append PR name. See example [here](https://github.com/Azure/azure-sdk-for-net/blob/176fd476588e8e61ebce747d66714cb39947cae5/sdk/azurestackhci/Azure.ResourceManager.Hci/src/autorest.md?plain=1#L52-L57).
- use `rename-mapping` to rename a model. See example [here](https://github.com/Azure/azure-sdk-for-net/blob/176fd476588e8e61ebce747d66714cb39947cae5/sdk/azurestackhci/Azure.ResourceManager.Hci/src/autorest.md?plain=1#L58-L76)

File: sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/EndpointProperties.cs [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+
+namespace Azure.ResourceManager.HybridConnectivity.Models
+{
+    /// <summary> Endpoint details. </summary>
+    public partial class EndpointProperties
```
Comment (archerzz) [inline sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/EndpointProperties.cs:14]: Name is too generic.

File: sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/HostType.cs [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.ResourceManager.HybridConnectivity.Models
+{
+    /// <summary> Enum of host cloud the public cloud connector is referencing. </summary>
+    public readonly partial struct HostType : IEquatable<HostType>
```
Comment (archerzz) [inline sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/HostType.cs:14]: Name is too generic.

File: sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/GenerateAwsTemplateContent.cs [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+            _serializedAdditionalRawData = serializedAdditionalRawData;
+        }
+
+        /// <summary> Initializes a new instance of <see cref="GenerateAwsTemplateContent"/> for deserialization. </summary>
+        internal GenerateAwsTemplateContent()
+        {
+        }
+
+        /// <summary> The name of public cloud connector. </summary>
+        public string ConnectorId { get; }
```
Comment (archerzz) [inline sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/GenerateAwsTemplateContent.cs:76]: Is it a GUID?

File: sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/IngressGatewayAsset.cs [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+            AccessKey = accessKey;
+            ExpiresOn = expiresOn;
+            ServiceConfigurationToken = serviceConfigurationToken;
+            _serializedAdditionalRawData = serializedAdditionalRawData;
+        }
+
+        /// <summary> The ingress hostname. </summary>
+        public string Hostname { get; }
+        /// <summary> The arc ingress gateway server app id. </summary>
+        public string ServerId { get; }
```
Comment (archerzz) [inline sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/IngressGatewayAsset.cs:81]: Is it a Guid?

File: sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/InventoryProperties.cs [hunk 9]
```diff
@@ -0,0 +1,85 @@
...
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+
+namespace Azure.ResourceManager.HybridConnectivity.Models
+{
+    /// <summary> Definition of inventory. </summary>
+    public partial class InventoryProperties
```
Comment (archerzz) [inline sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/InventoryProperties.cs:14]: Name is too generic.

File: sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/InventoryProperties.cs [hunk 10]
```diff
@@ -0,0 +1,85 @@
...
+            Status = status;
+            StatusDetails = statusDetails;
+            ProvisioningState = provisioningState;
+            _serializedAdditionalRawData = serializedAdditionalRawData;
+        }
+
+        /// <summary> Gets or sets the cloud native resource type. </summary>
+        public CloudNativeType? CloudNativeType { get; set; }
+        /// <summary> Gets or sets the cloud native resource name. </summary>
+        public string CloudNativeResourceId { get; set; }
```
Comment (archerzz) [inline sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/InventoryProperties.cs:75]: What's the format of this property value?
- If it's a GUID, use type [Guid](https://learn.microsoft.com/en-us/dotnet/api/system.guid?view=net-6.0). See example [here](https://github.com/Azure/azure-sdk-for-net/blob/bd953fc7a97bf617c892ce6314b50126b4021a4f/sdk/maps/Azure.ResourceManager.Maps/src/autorest.md?plain=1#L25).
- If it's a resource identifier, use type [ResourceIdentifier](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/ResourceIdentifier.cs). See example [here](https://github.com/Azure/azure-sdk-for-net/blob/bd953fc7a97bf617c892ce6314b50126b4021a4f/sdk/search/Azure.Search.Documents/src/autorest.md?plain=1#L106-L115).

File: sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/InventoryProperties.cs [hunk 11]
```diff
@@ -0,0 +1,85 @@
...
+        /// <summary> Gets or sets the mapped azure resource id. </summary>
+        public string AzureResourceId { get; set; }
```
Comment (archerzz) [inline sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/InventoryProperties.cs:77]: same

File: sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/ProvisioningState.cs [hunk 12]
```diff
@@ -0,0 +1,60 @@
...
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.ResourceManager.HybridConnectivity.Models
+{
+    /// <summary> The resource provisioning state. </summary>
+    public readonly partial struct ProvisioningState : IEquatable<ProvisioningState>
```
Comment (archerzz) [inline sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/ProvisioningState.cs:14]: Name is too generic

File: sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/ResourceProvisioningState.cs [hunk 13]
```diff
@@ -0,0 +1,54 @@
...
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.ResourceManager.HybridConnectivity.Models
+{
+    /// <summary> The provisioning state of a resource type. </summary>
+    public readonly partial struct ResourceProvisioningState : IEquatable<ResourceProvisioningState>
```
Comment (archerzz) [inline sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/ResourceProvisioningState.cs:14]: Name is too generic

File: sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/PublicCloudConnectorProperties.cs [hunk 14]
```diff
@@ -0,0 +1,89 @@
...
+        }
+
+        /// <summary> Cloud profile for AWS. </summary>
+        public AwsCloudProfile AwsCloudProfile { get; set; }
+        /// <summary> Host cloud the public cloud connector. </summary>
+        public HostType HostType { get; set; }
+        /// <summary> The resource provisioning state. </summary>
+        public ResourceProvisioningState? ProvisioningState { get; }
+        /// <summary> Connector primary identifier. </summary>
+        public string ConnectorPrimaryIdentifier { get; }
```
Comment (archerzz) [inline sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/PublicCloudConnectorProperties.cs:87]: Is it a GUID?

File: sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/ServiceName.cs [hunk 15]
```diff
@@ -0,0 +1,51 @@
...
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.ResourceManager.HybridConnectivity.Models
+{
+    /// <summary> Name of the service. </summary>
+    public readonly partial struct ServiceName : IEquatable<ServiceName>
```
Comment (archerzz) [inline sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/ServiceName.cs:14]: Name is too generic

**Review by archerzz — CHANGES_REQUESTED** [review]
Just a kind reminder. If you are going to release a new version, don't forget to update `CHANGELOG.md`. See [here](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process?anchor=signing-and-publishing-nuget-packges) for details.

File: sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/ServiceConfigurationResourceData.cs (hunk 1 above)
Comment (alexmahonic) [reply sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/ServiceConfigurationResourceData.cs:81]: Transformed to resource identifier for ResourceId

File: sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/AwsCloudProfile.cs (hunk 2 above)
Comment (alexmahonic) [reply sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/AwsCloudProfile.cs:78]: No, it's AWS accountId in 12-digit number

File: sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/GenerateAwsTemplateContent.cs (hunk 7 above)
Comment (alexmahonic) [reply sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/GenerateAwsTemplateContent.cs:76]: It's arm-id, transformed to resource identifier

File: sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/InventoryProperties.cs (hunk 11 above)
Comment (alexmahonic) [reply sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/InventoryProperties.cs:77]: It's arm-id, transformed to resource identifier type

File: sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/PublicCloudConnectorProperties.cs (hunk 14 above)
Comment (alexmahonic) [reply sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/PublicCloudConnectorProperties.cs:87]: Yes, transformed it to guid.

File: sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/InventoryResource.cs (hunk 3 above)
Comment (alexmahonic) [reply sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/InventoryResource.cs:23]: Added to prepend-rp-prefix

File: sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/CloudNativeType.cs (hunk 4 above)
Comment (alexmahonic) [reply sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/CloudNativeType.cs:14]: Added to prepend-rp-prefix

File: sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/EndpointProperties.cs (hunk 5 above)
Comment (alexmahonic) [reply sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/EndpointProperties.cs:14]: Added to prepend-rp-prefix

File: sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/HostType.cs (hunk 6 above)
Comment (alexmahonic) [reply sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/HostType.cs:14]: Added to prepend-rp-prefix

File: sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/ProvisioningState.cs (hunk 12 above)
Comment (alexmahonic) [reply sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/ProvisioningState.cs:14]: Added to prepend-rp-prefix

File: sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/ResourceProvisioningState.cs (hunk 13 above)
Comment (alexmahonic) [reply sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/ResourceProvisioningState.cs:14]: Added to prepend-rp-prefix

File: sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/ServiceName.cs (hunk 15 above)
Comment (alexmahonic) [reply sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/ServiceName.cs:14]: Added to prepend-rp-prefix

File: sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/IngressGatewayAsset.cs (hunk 8 above)
Comment (alexmahonic) [reply sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/IngressGatewayAsset.cs:81]: Yes it is. Updated with UUID type

File: sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/README.md [hunk 16]
```diff
@@ -10,7 +10,7 @@ This library follows the [new Azure SDK guidelines](https://azure.github.io/azur
     - Better error-handling.
     - Support uniform telemetry across all languages.
 
-## Getting started 
+## Getting started
 
 ### Install the package
```
Comment (archerzz) [inline sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/README.md:16]: You need to remove the `--prerelease` option in the sample below, if you want to release an official version.

File: sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/HybridConnectivityEndpointProperties.cs [hunk 17]
```diff
@@ -0,0 +1,80 @@
...
+
+        /// <summary> Initializes a new instance of <see cref="HybridConnectivityEndpointProperties"/> for deserialization. </summary>
+        internal HybridConnectivityEndpointProperties()
+        {
+        }
+
+        /// <summary> The type of endpoint. </summary>
+        public EndpointType EndpointType { get; set; }
+        /// <summary> The resource Id of the connectivity endpoint (optional). </summary>
+        public string ResourceId { get; set; }
```
Comment (archerzz) [inline sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/HybridConnectivityEndpointProperties.cs:76]: Just a double check. Is this a `UUID` or `ResournceIdentifier`?

File: sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/IngressGatewayAsset.cs (hunk 8 above)
Comment (archerzz) [reply sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/IngressGatewayAsset.cs:81]: It's still `string`. Please double check.

**Review by archerzz — CHANGES_REQUESTED** [review]
You forgot to update the API signature. Run `eng\scripts\Export-API.ps1 {your_sdk}` to update it.

File: sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/HybridConnectivityEndpointProperties.cs (hunk 17 above)
Comment (alexmahonic) [reply sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/HybridConnectivityEndpointProperties.cs]: updated

**Review by jsquire — DISMISSED** [review]
Please revert changes to all areas not related to your package.   You should not be updating any other packages here.

**Review by archerzz — APPROVED** [review]

**Review by jsquire — APPROVED** [review]
Approving based on Dapeng's previous approval.   Forcing merge to unblock.

**Pull request merged**